### PR TITLE
Fix event documentation example

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Adjusts/fixes the event button example, as reported by @Petro31 on Discord in the #beta channel.

It also changes the event types, as at the time of writing, the Hue events didn't exist yet. Now they do, and Hue doesn't have a double press 😅 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by edenhaus — APPROVED** [review]
Thanks @frenck  :P

File: source/_integrations/event.markdown [hunk 1]
```diff
@@ -43,7 +43,7 @@ Besides the timestamp of the last event, the event entity also keeps track of th
 
 This allows you, for example, to trigger a different action when the button on a remote control is pressed once or twice, if your remote control is capable of emitting these different types of events.
 
-When combining that with the [choose action](/docs/scripts/#choose-a-group-of-actions) script, you can assign multiple different actions to a single event entity. In the following example, pressing the button on the remote once or twice will trigger a different scene:
+When combining that with the [choose action](/docs/scripts/#choose-a-group-of-actions) script, you can assign multiple different actions to a single event entity. In the following example, pressing the button on the remote short or long will trigger a different scene:
```
Comment (c0ffeeca7) [inline source/_integrations/event.markdown:46]:
```suggestion
When combining that with the [choose action](/docs/scripts/#choose-a-group-of-actions) script, you can assign multiple different actions to a single event entity. In the following example, a different scene will be triggered depending on whether you press or press and hold the button:
```
The MS Style Guide seems to use _press and hold_ / _select and hold_
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/press-and-hold

also google:
https://support.google.com/accessibility/android/answer/7650693?hl=en

Alternatively, Apple use the term long-press:
https://developer.apple.com/documentation/uikit/touches_presses_and_gestures/handling_uikit_gestures/handling_long-press_gestures

File: source/_integrations/event.markdown [hunk 2]
```diff
@@ -53,19 +53,19 @@ action:
...
           condition: state
           entity_id: event.hue_remote_control_on_button
           attribute: "event_type"
-          state: "single_press"
+          state: "short_release"
         sequence:
           - scene: scene.living_room_evening
       - conditions:
-        - alias: "Move"
+        - alias: "Scene for watching a movie if the button was pressed long"
```
Comment (c0ffeeca7) [inline source/_integrations/event.markdown:64]:
```suggestion
        - alias: "Scene for watching a movie if the button was pressed and held"
```

**Review by c0ffeeca7 — COMMENTED** [review]
Tiny terminology tweak

File: source/_integrations/event.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/event.markdown:46]: Not in this case, as it is a physical remote, and not an UI button.

As per Microsoft:

CleanShot 2023-08-02 at 09 44 16


This is a hardware button, so matches the guides.

File: source/_integrations/event.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/event.markdown:64]: This is matching the Hue wording, that is used in the example.

File: source/_integrations/event.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/_integrations/event.markdown:46]: The style guide refers to a physical button.
image

**Pull request merged**